index.html

```diff
@@ -205,9 +220,12 @@ <h2 class="contact-form-title">Get In Touch</h2>
 
       </div>
       <div class="form-inputs">
-        <input type="text" placeholder="Full name">
-        <input type="text" placeholder="Email address">
-        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <div class="form-input-names">
+          <input name="name" type="text" placeholder="First name" maxlength="30" required>
+          <input name="name" type="text" placeholder="last name" maxlength="30" required>
+        </div>
+        <input name="email" type="email" placeholder="Email address" maxlength="30" required>
+        <textarea name="message" id="messageID" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
         <button type="submit" class="getintouchbtn">Get in touch</button>
       </div>
```

Comment:
- [ ] Hey, kindly adjust the styling in your form to match the Figma design..
 **YOUR DESIGN**

![clear](https://user-images.githubusercontent.com/59397159/167253431-f49b4f4b-d250-4020-b611-1be95d52bf40.png)




**FIGMA DESIGN**


![FORMS](https://user-images.githubusercontent.com/59397159/167253382-35495ad7-55e0-4474-8a6f-4e2760a8dbfb.png)